howto/aar/configure.md

```diff
@@ -35,101 +33,116 @@ The second command returns the name of the generated offer, for example, `my-con
     juju switch <model containing ams>
     juju relate ams <offer name>
 
-## Register clients manually
+## Configuring AAR for the Appliance
 
-If you are running the Anbox Cloud Appliance, you must register the clients manually.
+If you are using the appliance, you must register the clients manually. Adding clients manually requires access to the machines hosting AMS and the AAR.
 
-Adding clients manually requires access to the machines hosting AMS and the AAR.
+Install the snap:
 
-### Import AAR certificate
+    sudo snap install --channel= <channel> aar
 
-The first step is to import the AAR certificate into every AMS instance that should have access to the AAR. You can find the AAR certificate at `/var/snap/aar/common/certs/server.crt` on the machine hosting the AAR. Copy the certificate to the AMS machine and import it with the following command:
+Add the suitable snap version instead of `<channel>`. For example, it could be `1.27/stable`. For a list of all snap versions available, you can run `snap info aar`.
```

Comment:
This reads a bit odd. Shouldn't we tell users what to do directly? e.g.:

```suggestion
Replace `<channel>` with a suitable snap version. For example, it could be `1.27/stable`. For a list of all snap versions available, you can run `snap info aar`.
```

Wdyt?